### PR TITLE
[MRG+1] DOC shapes and collections is fully SG compatible

### PR DESCRIPTION
**Pull request by NelleV**
This is the last part to make the folder shapes and collection fully sphinx-gallery compatible.

### CODE REVIEW

File: examples/shapes_and_collections/plot_path_patch.py [hunk 1]
```diff
@@ -2,6 +2,9 @@
 ================
 PathPatch object
 ================
+
+This example shows how to create Paths and PathPatch objects through
```
Comment (QuLogic) [inline examples/shapes_and_collections/plot_path_patch.py]: Backticks for cross-references? (Does that work here?)

Comment (anntzer) [inline examples/shapes_and_collections/plot_path_patch.py]:
```
`Path`\s
```
(see e.g. https://github.com/anntzer/mplcursors/blame/master/doc/index.rst#L126, rendered at https://mplcursors.readthedocs.io/en/stable/#customization ("Line2Ds"))

Comment (NelleV) [inline examples/shapes_and_collections/plot_path_patch.py]: backticks might work automatically for cross references. There's unfortunately no easy way to check this just yet, but we'll see once I merge this branch into the sphinx-gallery branch.

File: examples/shapes_and_collections/plot_path_patch.py [hunk 2]
```diff
@@ -2,6 +2,9 @@
 ================
 PathPatch object
 ================
+
+This example shows how to create `Path`\s and `PathPatch` objects through
+Matplotlib's API.
 """
```
Comment (phobson) [inline examples/shapes_and_collections/plot_path_patch.py]: do we need blank lines around this?

Comment (NelleV) [inline examples/shapes_and_collections/plot_path_patch.py]: I am not sure I understand the question. If you are asking about the need for blanklines around `"""`, then the answer is no.

Comment (phobson) [inline examples/shapes_and_collections/plot_path_patch.py]: OK sweet 👍

**Comment by anntzer** [comment]
Thanks!

**Pull request merged**